File: src/main/java/xyz/lncvrt/galaxyboxpvp/events/PlayerDropItemListener.java
package xyz.lncvrt.galaxyboxpvp.events;

import xyz.lncvrt.galaxyboxpvp.GalaxyBoxPvP;
import org.bukkit.event.EventHandler;
import org.bukkit.event.EventPriority;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerDropItemEvent;

public class PlayerDropItemListener implements Listener {
    private final GalaxyBoxPvP plugin;

    public PlayerDropItemListener(GalaxyBoxPvP plugin) {
        this.plugin = plugin;
    }

    @EventHandler(priority = EventPriority.LOWEST)
    public void onItemDrop(PlayerDropItemEvent event) {
        plugin.convertInventoryItemsPrep(event.getPlayer(), event.getPlayer().getUniqueId());
    }
}

File: src/main/kotlin/xyz/lncvrt/galaxyboxpvp/events/PlayerDropItemListener.kt
package xyz.lncvrt.galaxyboxpvp.events

import org.bukkit.event.EventHandler
import org.bukkit.event.EventPriority
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerDropItemEvent
import xyz.lncvrt.galaxyboxpvp.GalaxyBoxPvPPlugin

class PlayerDropItemListener(private val plugin: GalaxyBoxPvPPlugin) : Listener {
    @EventHandler(priority = EventPriority.LOWEST)
    fun onItemDrop(event: PlayerDropItemEvent) {
        plugin.convertInventoryItemsPrep(event.getPlayer(), event.getPlayer().uniqueId)
    }
}